### PR TITLE
Update OpenAPI and docs to use StatusList2021

### PR DESCRIPTION
**Pull request by rhofvendahl**
This PR updates interop docs and openapi to be compatible with the [Status List 2021 spec](https://www.w3.org/TR/vc-status-list/), as discussed in https://github.com/w3c-ccg/traceability-interop/pull/555 and mandated by the [current Trace Interop spec](https://w3c-ccg.github.io/traceability-interop/draft/#data-integrity-proof-suites).

### CODE REVIEW

**Review by mkhraisha — APPROVED** [review]
Lgtm, some comments about issues we will need to raise

**Review by mkhraisha — CHANGES_REQUESTED** [review]
There are still some references to RevocationList2020, for e.g. in [VerifiablePresentation.yml](https://github.com/mesur-io/traceability-interop/blob/7dbbd5a07fb5607086a6811fff2bca229c51445c/docs/openapi/schemas/VerifiablePresentation.yml#L37).

Also we are still importing contexts for revelationList2020 and not importing the context for StatusList2021, that should be included here, before we merge in any statusList PR otherwise the tests will fail

**Review by OR13 — APPROVED** [review]

File: docs/openapi/schemas/StatusListCredential.yml
Comment (OR13) [inline docs/openapi/schemas/StatusListCredential.yml:15]: 100% needed for v1 ... 100% an error for v2

File: docs/tutorials/credentials-revocation/README.md
Comment (OR13) [inline docs/tutorials/credentials-revocation/README.md:46]: this could have been a "new suite"... that probably would have been a smarter way to add support.

File: docs/openapi/resources/credential-status.yml
Comment (rhofvendahl) [reply docs/openapi/resources/credential-status.yml:54]: There are a lot of changes that'll have to be made for that transition, I can make an issue tracking that as you suggested

File: docs/openapi/schemas/StatusListCredential.yml
Comment (rhofvendahl) [reply docs/openapi/schemas/StatusListCredential.yml:28]: Updating that to the mesur production url

File: docs/openapi/schemas/StatusListCredential.yml
Comment (rhofvendahl) [reply docs/openapi/schemas/StatusListCredential.yml]: Just updated this to use the mesur prod api

File: docs/tutorials/credentials-revocation/README.md
Comment (rhofvendahl) [reply docs/tutorials/credentials-revocation/README.md]: Thanks, made those changes in [this](https://github.com/w3c-ccg/traceability-interop/pull/559) PR

File: docs/openapi/resources/credential-status.yml
Comment (rhofvendahl) [reply docs/openapi/resources/credential-status.yml:54]: Issue: https://github.com/w3c-ccg/traceability-interop/issues/565

File: docs/openapi/resources/credential-status.yml
Comment (rhofvendahl) [reply docs/openapi/resources/credential-status.yml:45]: Just created an issue: https://github.com/w3c-ccg/traceability-interop/issues/566

**Comment by TallTed** [comment]
I have no material comment. I just wanted to make sure everyone got to quickly giggle at the typo I just encountered above. revelationList2020. It suddenly puts that document into a very different category....

**Review by OR13 — APPROVED** [review]

**Review by nissimsan — APPROVED** [review]
+1 on eggs breaking

**Comment by nissimsan** [comment]
@rhofvendahl , pls merge if you can get @mkhraisha's approval on this.

**Review by mkhraisha — APPROVED** [review]

**Comment by mkhraisha** [comment]
Lgtm! 
And LOL at revelations list

**Comment by mkhraisha** [comment]
@rhofvendahl Please feel free to merge.

**Pull request merged**
